File: Python/Pattern_d6.py
print("Enter the number of rows")
n=int(input())

print("Here is the pattern")

i=n
while i>=1:
    k=n
    for j in range(1,i+1):
        print(k,end=" ")
        j+=1
        k-=1
    print("\n")
    i-=1

File: patternsJAVA/patternD6.java
/*
Pattern is:
4 3 2 1
4 3 2
4 3 
4
*/
public class patternD6 {
    public static void main(String[] args) {
        int n=4;
        for (int i = 1; i <= n; i++) {
            for (int j = n; j >=i; j--) {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
